apps/prairielearn/src/components/NavbarContext.html.ts

```diff
@@ -140,6 +140,12 @@ const navPagesTabs: Partial<Record<Exclude<NavPage, undefined>, TabInfo[]>> = {
       iconClasses: 'fas fa-quote-right',
       tabLabel: 'Topics',
     },
+    {
+      activeSubPage: 'modules',
+      urlSuffix: '/course_admin/modules',
+      iconClasses: 'fa fa-layer-group',
```

Comment:
I used the `fa-layer-group` icon but was unsure if this is the best fit. Let me know if you have other suggestions for this icon. 
![Screenshot 2024-09-13 at 2 22 35 PM](https://github.com/user-attachments/assets/220fc7ee-c662-4056-b418-20c238f4a689)

Reply:
I like this icon!